packages/rulesets/src/spectral/functions/get-collection-only-has-value-nextlink.ts

```diff
@@ -1,17 +1,27 @@
 // Verifies that collection get schemas have only the `value` and `nextLink` properties in their models.
 
-export const getCollectionOnlyHasValueAndNextLink = (properties: any, _opts: any, ctx: any) => {
-  if (!properties || typeof properties !== "object") {
-    return []
-  }
-  const keys = Object.keys(properties)
-
-  if (keys.length != 2 || !keys.includes("value") || !keys.includes("nextLink")) {
-    return [
-      {
-        message: "Get endpoints for collections of resources must only have the `value` and `nextLink` properties in their model.",
-      },
-    ]
+export const getCollectionOnlyHasValueAndNextLink = (prop: any, _opts: any, ctx: any) => {
+  for (let path of ctx.path) {
+    if (path.includes(".")) {
+      // Splitting the path by Provider namespace
+      let splitNamespace = path.split(".")
```

Comment:
> let splitNamespace = path.split(".") [](http://example.com/codeflow?start=5&length=37)

how are you extracting just the part of the path after namespace?

Reply:
Namespace will have dot right, Microsoft.Namespace so with I am dividing the path.

Reply:
ok, sorry I didnt read the code properly before. Thinking about this it is possible that some paths have 2 or more namespace names in them, and we should always apply the logic for the last namespace name in the path. CAn u adjust the logic to account for this?

Reply:
Added logic to cover if there are multiple provider namespaces.